### PR TITLE
Wording error in description of HTML waypoint

### LINKED ISSUE #2779: Minor Wording in 'Use Hex Code to Color Elements Red'
Challenge http://www.freecodecamp.com/challenges/waypoint-use-hex-code-to-color-elements-red has an issue. 

In the sentence: "16 colors and 6 positions means we have 16 to the 6th power, or more than 16 million possible colors," it would be more accurate to say "16 values" instead of "16 colors."

### PR DESCRIPTION
**Pull request by butterthebuddha**
Changed description from `16 colors` to  `16 values`
closes #2779

### CODE REVIEW

File: seed/challenges/html5-and-css.json [hunk 1]
```diff
@@ -3507,7 +3507,7 @@
       "difficulty": 1.56,
       "description": [
         "You may be wondering why we use 6 digits to represent a color instead of just one or two. The answer is that using 6 digits gives us a huge variety.",
```
Comment (abhisekp) [inline seed/challenges/html5-and-css.json]: You didn't correct this https://github.com/FreeCodeCamp/FreeCodeCamp/issues/2779#issuecomment-138092681

**Comment by awesomeaniruddh** [comment]
6 digits loos OK to me

**Comment by abhisekp** [comment]
@awesomeaniruddh They're not **digits**. 
They can be either be called 6 characters or 6 positions.

**Comment by awesomeaniruddh** [comment]
idk, I don't see why they can't be called digits?

**Comment by BerkeleyTrue** [comment]
Its still digits for hex.

**Comment by abhisekp** [comment]
@BerkeleyTrue @awesomeaniruddh But I don't see any **"digits"** in the color `#AFEDFF` :confused:

**Comment by awesomeaniruddh** [comment]
AFEDFF is a hexadecimal number. Because our standard number system has only 10 digits, and hexadecimal needs 16, letters from A-F serve as *digits* in the hexadecimal number system.

**Comment by abhisekp** [comment]
@awesomeaniruddh ok. That explain it. :smiley:

**Pull request merged**